app/src/main/java/de/danoeh/antennapod/fragment/preferences/UserInterfacePreferencesFragment.java

```diff
@@ -70,7 +76,13 @@ private void setupInterfaceScreen() {
                     }
                     final Context context = getActivity();
                     final String[] navTitles = context.getResources().getStringArray(R.array.back_button_go_to_pages);
-                    final String[] navTags = context.getResources().getStringArray(R.array.back_button_go_to_pages_tags);
+                    final String[] navTags = new String[]{
+                            HomeFragment. TAG,
+                            InboxFragment.TAG,
+                            QueueFragment.TAG,
+                            EpisodesFragment.TAG,
+                            SubscriptionFragment.TAG
+                    };
```

Comment:
How about splitting the back button behavior setting to a "Default page" setting (that is the home screen by default) that we always go back to and a "confirm closing app" setting?

Reply:
Isnt that already in the back button preferences?